modules/veteran_verification/app/controllers/veteran_verification/v0/keys_controller.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'base64'
+require_dependency 'notary'
+
+module VeteranVerification
+  module V0
+    # KeysController returns a set of public keys as Base64URL encoded PEMs
+    # that can be matched based on the kid (key id) and used to verify
+    # payloads signed by the same Notary in other endpoints.
+    class KeysController < ApplicationController
+      skip_before_action(:authenticate)
+
+      NOTARY = VeteranVerification::Notary.new(Settings.vet_verification.key_path)
+
+      def index
+        render json: {
+          keys: [
+            {
+              kty: 'RSA',
+              use: 'sig',
+              alg: 'RS256',
+              kid: NOTARY.kid,
+              pem: Base64.urlsafe_encode64(NOTARY.public_key.to_pem)
```

Comment:
No good answer for you here other than it fits with how other fields are handled in specs that we're currently not complying with 🙃

Reply:
That makes sense. I would argue that if we keep this field here we shouldn't base64 encode it, since the main purpose is to make this usable for devs who haven't read the entire JWK spec and aren't using a library that parses a JWK automatically.

Reply:
Took out the base64 encoding 👍